rskj-core/src/test/java/co/rsk/peg/feeperkb/constants/FeePerKbConstantsTest.java

```diff
@@ -0,0 +1,98 @@
+package co.rsk.peg.feeperkb.constants;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.peg.vote.AddressBasedAuthorizer;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.crypto.ECKey;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FeePerKbConstantsTest {
+
+    @ParameterizedTest
+    @MethodSource("getGenesisFeePerKbTestProvider")
+    void getGenesisFeePerKbTest(FeePerKbConstants feePerKbConstants, Coin expectedGenesisFeePerKb) {
+        Coin actualGenesisFeePerKb = feePerKbConstants.getGenesisFeePerKb();
+        assertEquals(expectedGenesisFeePerKb, actualGenesisFeePerKb);
+    }
+
+    private static Stream<Arguments> getGenesisFeePerKbTestProvider(){
+        return Stream.of(
+                Arguments.of(FeePerKbMainNetConstants.getInstance(), Coin.MILLICOIN.multiply(5)),
```

Comment:
Check the indent size in your intellij config

<img width="552" alt="image" src="https://github.com/rsksmart/rskj/assets/53787863/a7d9d82f-8d5d-40b6-8f5f-158e11e6bde1">

Reply:
I just applied this change! I don't know why I lost this setup when I created a new branch.

<img width="976" alt="image" src="https://github.com/rsksmart/rskj/assets/160710714/35518ba3-d4f5-4454-b1b4-a94343d0cca4">